### ISSUE TITLE
Duplicate notification email / duplicate email in notification targets list

### ISSUE THREAD

**Issue by RichGriese** [vector-im/element-web#11142]
I've got duplications of the same email address, one with capitalization and one without in the notifications list. I'm getting two email notifications. Sweet baby jesus help me.

**Comment by t3chguy** [comment]
e-mail addresses by design are case sensitive, in Settings > Notifications you can disable notifications for the one you wish to remove, and you can remove it in Settings > General

**Comment by RichGriese** [comment]
@t3chguy Thanks for the response. The "capitalization" thing was just a hunch. Prior to my issue post I had tried, removing me gmail in settings > General and adding it back. And playing with the Setting > Notification slider. But it hadn't helped. Here's some more info.

I went to settings > general and removed my email address. I've done that before and I know when I add it again it's gonna ask me to confirm, so I haven't added anything back yet. When I go over to settings > notifications there email notification slider says that I need to add an email address to get notifications. Yet... down in the list of notifications targets it still list both the email address. I've also gone to the settings > help / about and cleared the cache, and those two email addresses still show.

I think this issues is... how do I removed items from the settings > notifications > notifications targets?

BTW... I'm on a mac, using the web interface.

**Comment by t3chguy** [comment]
Add the e-mail address (with matching capitalisation) then verify
then in Settings > Notifications you can toggle it off (Enable e-mail notifications <...>):

![image](https://user-images.githubusercontent.com/2403652/66832959-d3b93d80-ef52-11e9-8333-55c0625586a0.png)

**Comment by RichGriese** [comment]
Ok... yeah... I tried this before too. so, I've added back my email address to the account, and and can toggle on and off the notification to the one that is capitalized. And when I do, the capitalized one shows and disappears. but when it's off and there is only one email showing... I don't get any email notifications, and if I put it on and it is showing, I get 2 email notifications.

It seems either their should me a enable email notifications slider for each email in the top notifications on/off area, or I in some way got an extra email in the emails targets list at the bottom of that area.

**Comment by RichGriese** [comment]
Also just to be clear... in the preferences > general there is only one email address associated with the account. I just added it with caps before, and it now lists as all lower case. Yet... over on the preferences > notifications area it still lists both the capitalized and non-capitalized in the notifications targets, but there is only one slider to turn turn on / off notifications and it is all lower case.

**Comment by RichGriese** [comment]
I'm suspecting there is a bug in the routine to add items to the notifications targets and that it's adding items to that set/list without checking for capitalization, and making it impossible to latere remove the duplicate. But... that's just a half ass hunch. But, for me... I just wanna have someone explain how to move to getting only one email notification :)

**Comment by RichGriese** [comment]
Here's another thought. The issue may be in the initial sign up routine? Looking back, I originally signed up with my email address, and looking, I added it with capitalization. That might be where that item got into the preferences > notifications > target's list? Then when I started using the notifications, I added the email again, it converted it to lower case, and it added a 2nd record? Point is... not matter when I do, I can't seem to get rid of the notifications lists items with capitalization item. Even if I turn off notifications and eliminate emails from my account, that capitalized email still shows in the notifications items list at the bottom of the notifications preference panel.

**Comment by RichGriese** [comment]
The journey continues... after eliminating my email again, and logging out... I am now unable to log back in. when I try to log in with my user name and password I get the message "Unable to connect to Homeserver. Retrying..." and if I try logging in with my email address I get "incorrect user name and password" My account may be fucked. 

But... on the progress side... I did just get a SINGLE email alert for activity to a room I am in. So, in that sense I've moved forward.

So... will now get email updates... but not be able to ever log into the account again?

Can't decide if I should listen to the Beatles song, The Long & Winding Road https://www.youtube.com/watch?v=fR4HjTH_fTM, or the Dead song Truckin' (what a long strange trip it's been) https://www.youtube.com/watch?v=pafY6sZt0FE
